syntax = "proto3";

package payment;

service PaymentService {
  rpc PayOrder (PaymentRequest) returns (PaymentResponse);
}

message PaymentRequest {
  string order_id = 1;
  int32 amount = 2;
}

message PaymentResponse {
  bool success = 1;
  string message = 2;
}
